packages/api-observable/src/Calls.ts

```diff
@@ -47,4 +47,8 @@ export default class ApiCalls extends ApiQueries {
   submitExtrinsic = (extrinsic: Extrinsic): Observable<Hash | undefined> => {
     return this._api.author.submitExtrinsic(extrinsic);
   }
+
+  submitAndWatchExtrinsic = (extrinsic: Extrinsic): Observable<ExtrinsicStatus | undefined> => {
```

Comment:
The Observable can also fire`undefined`?

Reply:
Actually I believe not anymore. It used to in certain error conditions. I just have not cleaned-up up any of these and need to re-visit. I kept this one for consistency with the others.